metering/metering-installing-metering.adoc

```diff
@@ -5,41 +5,41 @@ include::modules/common-attributes.adoc[]
 
 toc::[]
 
-On {product-title} 4 clusters the Metering Operator can be installed through the OperatorHub. If you are using an earlier version of OpenShift or a Kubernetes cluster without OpenShift, use the link:https://github.com/operator-framework/operator-metering/tree/master/Documentation[upstream documentation] to install, configure, and use the Metering Operator.
+On {product-title} 4 clusters metering can be installed through OperatorHub. If you are using an earlier version of {product-title} or a Kubernetes cluster without OpenShift, use the link:https://github.com/operator-framework/operator-metering/tree/master/Documentation[upstream documentation] to install, configure, and use metering.
+
+To get started installing metering, first install the Metering Operator from OperatorHub. Next, configure your instance of metering by creating a `CustomReousrceDefinition` (CRD), referred to here as your `MeteringConfig`. The installation process creates a default `MeteringConfig` that you can modify using the examples in the documentation. After creating your `MeteringConfig`, install the metering stack. Last, verify your installation.
 
 include::modules/metering-install-operator.adoc[leveloffset=+1]
 
 // This content is included directly in the assembly and NOT in a module because the workflow for installing metering requires linking off to the config docs, and xrefs are not allowed in modules. [klamenzo]
 [id="metering-install-metering-stack_{context}"]
 == Installing the Metering stack
 
-To install the metering stack, create a `MeteringConfig` resource and add your configuration details.
+To install the metering stack, first review the xref:../metering/configuring-metering/metering-about-configuring.adoc#metering-about-configuring[configuration options] and create a `MeteringConfig` resource. You can begin the process below to generate a default `MeteringConfig`, then use the examples in the documentation to modify this default file for your specific installation. 
```

Comment:
What if you include this information as a heading in the assembly called _About installing the metering stack_ or _before installing the metering stack_. Then a separate module for _Installing the metering stack_??

Reply:
I can modularize this now that we've removed the restriction on links in modules. I think a link is important here for any customer coming directly to this point in docs (via search or link, etc.) In my opinion the workflow for metering is a big clunky here, because you're provided a default config file in the UI, but you have to go off and review and understand the necessary config options before you can proceed. I think having a link helps readers switch easily to a new context, the config information.